web/controllers/admin_controller.ex

```diff
@@ -319,6 +319,7 @@ defmodule ExAdmin.AdminController do
   end
 
   defp to_integer(:string, string), do: string
+  defp to_integer(:id, string), do: string
```

Comment:
Dev, 

I welcome contributors/contributions to ex_admin. It a pretty large package and I can use help in features, bug fix, docs, usability, and general feedback. 

As for converting the field to an integer or not, two test cases will provide that. Since you are already using an integer primary key, then I can presume that conversion is not necessary. 

I'm trying to use the policy moving forward to add tests for each fix. I did not write any tests with the original development, so adding tests for each fix is my way of growing the test suite. 

So, it you can add a couple tests for this fix, I would be grateful. 

Thanks...Steve